src/main/java/lotto/Input.java

```diff
@@ -0,0 +1,79 @@
+package lotto;
+
+import lotto.CustomException.WrongInputNumberException;
+
+import java.util.Scanner;
+
+public class Input {
+    private static Scanner scanner = new Scanner(System.in);
+    private static String numberRegularExpression = "^[0-9]+$";
+    public static int inputMoney(){
+        System.out.println("구입금액을 입력해 주세요.");
+        String inputValue = scanner.nextLine();
+        return convertToInteger(inputValue);
+    }
+
+    private static int convertToInteger(String inputValue){
+        String numberRegular_Expression = "^[0-9]+$";
+        if(!inputValue.matches(numberRegular_Expression)) throw new WrongInputNumberException("숫자를 입력해주세요.");
+        return Integer.parseInt(inputValue);
+    }
+
+    public static Integer[] inputPreviousLottoNumber(){
+        System.out.println("지난 주 당첨 번호를 입력해 주세요.");
+        Integer[] prevLottoNumber = prevLottoNumberSplit(scanner.nextLine());
+        previousLottoNumberValidation(prevLottoNumber);
+        return prevLottoNumber;
+    }
+
+    public static Integer[] prevLottoNumberSplit(String previousLottoNumber){
+        Integer[] integerNumber = new Integer[6];
+        previousLottoNumber = previousLottoNumber.replace(" ","");
+        String[] splitNumber = previousLottoNumber.split(",");
+        for(int i =0;i<splitNumber.length;i++){
+            integerNumber[i] = Integer.parseInt(splitNumber[i]);
+        }
+        return integerNumber;
+    }
+
+    private static void previousLottoNumberValidation(Integer[] previousLottoNumber){
+        checkPreviousLottoNumberLength(previousLottoNumber);
+        for(int i=0;i<previousLottoNumber.length;i++) {
```

Comment:
for는 다른 방법도 하나 있어요.
```suggestion
        for(int lottoNumber : previousLottoNumber) {
```
JS에서도 있었던 것 같은데 길이를 호출하는 것보다 이런 방식이 더 좋겠죠?
아래에 있는 친구들도 한번 수정하는 것을 생각해보죠.